packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-time-picker/gux-form-field-time-picker.tsx

```diff
@@ -74,10 +86,37 @@ export class GuxFormFieldTimePicker {
 
   @OnMutation({ childList: true, subtree: true })
   onMutation(): void {
+    this.labelInfo = this.root.querySelector('gux-label-info-beta');
     this.hasError = hasSlot(this.root, 'error');
     this.hasHelp = hasSlot(this.root, 'help');
   }
 
+  @Listen('keyup')
+  handleKeyup(event: KeyboardEvent): void {
+    switch (event.key) {
+      case 'Tab': {
+        if (this.input.matches(':focus-visible')) {
+          void this.labelInfo?.showTooltip();
+          setTimeout(() => {
+            void this.labelInfo?.hideTooltip();
+          }, 6000);
```

Comment:
why is it that we wait 6 seconds here?

Reply:
What's the reasoning behind saving the timeout into `hideLabelInfoTimeout` and then clearing that timeout later? This is more for my own understanding.

Reply:
Looks like 6000 is hardcode in several files, so if UX asks to change that value we'll need to change it in multiple places. If you don't think they'll ever ask to change it then no worries, just wanted to mention this and see what you thought.

Reply:
> What's the reasoning behind saving the timeout into `hideLabelInfoTimeout` and then clearing that timeout later? This is more for my own understanding.

On this one, I did some testing. It's to avoid this scenario:
- I tab onto field and tooltip displays, set timeout starts counting
- I tab into next field, timeout still counting
- I tab back to tooltip field, tooltip shows but it hidden abruptly by the hideTooltip coming from set timeout
By clearing the timeout each time the tooltip is dismissed manually, it stops the timed hide tooltip from interrupting later. It's a pretty niche issue but I think good to avoid. 

> Looks like 6000 is hardcode in several files, so if UX asks to change that value we'll need to change it in multiple places. If you don't think they'll ever ask to change it then no worries, just wanted to mention this and see what you thought.
I think it's a good thing to raise, there's a lot of duplication in these files so if this PR has taught me anything is that any changes we make in form fields will require a lot of changes 😄. My take is that this doesn't make it any worse at this point. That ok?